ext/hash/bench.php

```diff
@@ -3,7 +3,8 @@
 
 error_reporting(E_ALL);
 
-$data = file_get_contents(__FILE__);
+// 2240 comes from the fact that it used to be file_get_contents(__FILE__);
+$data = str_repeat("\x00", 2240);
```

Comment:
It's been constant before, too?

Reply:
@m6w6 not exactly. 
in revision 7a0886509ef0edb62a452f24387a67fb4bf175e5 it was 1949 bytes.
in revision 8d3f8ca12a0b00f2a74a27424790222536235502 it was 1938 bytes.
in revision 02294f0c84740a2c3fd2d13a4a66544717d778bb it was 1957 bytes.
in revision 58b17906f512866c2e34844fa497ecdf7f1e1e3d it was 2178 bytes
in revision 3d1e7d37ebd80698c11923dcf049e75dbe9422d3 it was 2240 bytes
, and in the current revision it's 503 bytes.. ( e0ea3e8a0180ec86e3c5da80e4a0bf5b12dea84d )

Reply:
I don't think there's any point in preserving the exact previous size. Let's make this a nice round number like 2048. Could also make it `$argv[1] ?? 2048` to allow passing a different size :)

Reply:
@nikic sounds good to me, how about https://github.com/php/php-src/pull/6386/commits/e213df9934c5242c24b704e058fc31f755957fee